packages/cli/src/lib/fs-utils.js

```diff
@@ -0,0 +1,33 @@
+import fs from 'node:fs/promises';
+
+// Small helper to retry copyFile on transient errors (EBUSY on Windows CI).
+// Attempts: number of tries (default 5)
+// baseDelay: starting delay in ms for exponential backoff (default 100ms)
+async function copyFileWithRetry(source, target, { attempts = 5, baseDelay = 100 } = {}) {
+  let lastError;
+
+  for (let i = 0; i < attempts; i++) {
+    try {
+      return await fs.copyFile(source, target);
+    } catch (err) {
+      lastError = err;
+
+      // Only retry for transient EBUSY / EPERM on Windows or generic resource busy errors
+      if ((err.code === 'EBUSY' || err.code === 'EPERM' || err.code === 'EACCES') && i < attempts - 1) {
+        const delay = baseDelay * Math.pow(2, i);
+        // add small jitter
```

Comment:
I’m curious, why would jitter be useful here? It seems cool, but when I actually think about it I can’t think of an actual use case for it